scripts/submit_jobs.py

```diff
@@ -12,7 +12,8 @@
 # current data global tag
 CONDITIONS = '101X_dataRun2_HLT_v7'
 # L1 calibrations; needs to be updated when L1 calibrations change
-CALOSTAGE2PARAMS = '2018_v1_1_ECALZS'
+#CALOSTAGE2PARAMS = '2018_v1_1_ECALZS'
```

Comment:
Please remove this commented line. We will likely never use this tag again and so there's no need to keep it here. If someone really wants to know the history of the script, they can look at the revision log.